### ISSUE TITLE
GitHub download is very slow - Upload current version also on other webpage (maybe on crewl.ink?)

### ISSUE THREAD

**Issue by SirBuvex** [ottomated/CrewLink#540]
# Pre-Flight Checklist
Please use this checklist to avoid spamming:

1. [x] I am not asking a question => use the [Discord](https://discord.gg/9mwuVNA) if you are
2. [x] I have checked that someone else has not suggested this using the [search bar](https://github.com/ottomated/CrewLink/issues?q=is%3Aissue)
3. [x] My feature request is not one of the **commonly suggested features**:
- Adjustable voice radius
- Mobile support
- Linux / MacOS support
- Itch.io support

**Is your feature request related to a problem? Please describe.**
Downloads on GitHub are very slow sometimes! CrewLink tries to update for over an hour now and still downloaded only 12MB! I did a Speedtest and my normal download speed is over 60 Mbps.

**Describe the solution you'd like**
It would be great if there was another (additional) place where the CrewLink client (and other users) can download the latest version (maybe on crewl.ink?)

**Describe alternatives you've considered**
_empty_

**Additional context**
_empty_

**Comment by 231187** [comment]
It took me about 2 seconds to download the entire setup. Using 1g up and down.

**Comment by SirBuvex** [comment]
> 
> 
> It took me about 2 seconds to download the entire setup. Using 1g up and down.

Yes, I tried it again now and everything was also fine. But friends had also this issue... 
It seems that the GitHub servers are just sometimes so slow.

**Comment by Ich5003** [comment]
It's not about the Github servers, github uses Amazon S3 buckts, and there are some internet providers (e.g. Telekom Germany), which have a bad peering to those which results in slow downloads from Amazon S3

**Comment by SirBuvex** [comment]
> It's not about the Github servers, github uses Amazon S3 buckts, and there are some internet providers (e.g. Telekom Germany), which have a bad peering to those which results in slow downloads from Amazon S3

Thank you! Than I know where the problem results from 😉

Nevertheless it would be great if the problem could be circumvented with an additional download option.
